tests/sdl/whitelists.py

```diff
@@ -114,6 +114,7 @@ class OvmsBaseImageType(Enum):
         'libtbb2',
         'libxml2',
         'openssl',
+        'libbrotli1'
```

Comment:
'libtrotli1' should be included in packages[OvmsBaseImageType.UBUNTU_NGINX] only

Reply:
@ngrozae that would not be correct. libbrotli1 is a dependency of curl which are are adding the the release image regardless of the nginx